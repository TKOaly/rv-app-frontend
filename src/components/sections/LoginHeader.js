import './styles/LoginHeader.scss';
import React from 'react';

const LoginHeader = () => (
    <div className="loginHeader">
        <h1>Ruokav√§litys</h1>
    </div>
);

export default LoginHeader;
