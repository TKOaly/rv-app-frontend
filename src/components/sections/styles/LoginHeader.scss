@import "vars";

header {
  padding-top: 4px;
  padding-bottom: 4px;
  vertical-align: middle;
  box-sizing: border-box;
  -webkit-box-sizing: border-box;
  background-color: $header-bgcolor;
  display: flex;
  flex-direction: row;
  align-items: center;
  border-bottom: 2px solid #fc3;
  margin: 0 16px;

  .header-title {
    display: flex;
    flex-direction: row;
    align-items: center;
    margin-bottom: -7px;

    h1 {
      font-size: 1.6em;
      font-weight: normal;
      color: #fff;
    }

    img {
      margin-left: -22px;
      width: 100px;
      height: auto;
    }
  }
}
