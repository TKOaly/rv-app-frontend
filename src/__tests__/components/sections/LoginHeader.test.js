import LoginHeader from '../../../components/sections/LoginHeader';
import React from 'react';
import renderer from 'react-test-renderer';

describe('<LoginHeader/>', () => {
    it('should match snapshot', () => {
        const cmpnt = renderer.create(<LoginHeader />).toJSON();
        expect(cmpnt).toMatchSnapshot();
    });
});

