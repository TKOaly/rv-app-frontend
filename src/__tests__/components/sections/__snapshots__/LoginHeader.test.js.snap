// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoginHeader/> should match snapshot 1`] = `
<header>
  <div
    className="header-title"
  >
    <img
      alt="logo"
      src="tkoaly2.svg"
    />
    <h1>
      Ruokav√§litys
    </h1>
  </div>
</header>
`;
